kartothek/io/dask/delayed.py

```diff
@@ -156,6 +156,7 @@ def merge_datasets_as_delayed(
     match_how="exact",
     label_merger=None,
     metadata_merger=None,
+    secondary_indices=None,
```

Comment:
Actually, I thought about this pipeline again and I believe we don *not* want to allow index building here. Index building should only be supported for pipelines with immediate write which is not the case here. Sorry for the confusion. Can you please remove this again?

Reply:
Sure